ci/purge_expired_sessions_from_datastore.yaml

```diff
@@ -0,0 +1,44 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  PROJECT_ID:
+  DATAFLOW_TEMPLATE_VERSION:
+  EXPIRATION_TIME_OFFSET:
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=/root/gcloud-service-key.json
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+
+      gcloud auth activate-service-account --key-file "${GOOGLE_APPLICATION_CREDENTIALS}"
+      gcloud config set project "${PROJECT_ID}"
+
+      DATAFLOW_JOB=$(gcloud dataflow jobs run purgesession \
```

Comment:
The job name should be snake case or pascal case.

Reply:
D'oh, I think I was actually getting at kebab-case but oh well, better than one previous so 👍 .